styx-cli/src/main/java/com/spotify/styx/cli/Main.java

```diff
@@ -415,6 +415,9 @@ private void triggerWorkflowInstance()
     final WorkflowInstance workflowInstance = getWorkflowInstance(namespace);
     final ByteString payload = serialize(workflowInstance);
     send(Request.forUri(apiUrl("scheduler", "trigger"), "POST").withPayload(payload));
+    cliOutput.printMessage(CliOutput.CliMessage.create(
```

Comment:
Thought about this actually. How to make it generic enough for other cases. How about:

```
{
   "type":"info/warning/error",
   "message":"...",
   "details":"..."
}
```

In this case, `details` may contain workflow information.